src/sccp_socket.c

```diff
@@ -457,11 +491,12 @@ static int sccp_read_data(sccp_session_t * s, sccp_msg_t * msg)
 
 	errno = 0;
 	int tries = 0;
-	int max_retries = 30;											/* arbitrairy number of tries to read a message */
+	int max_retries = READ_RETRIES;											/* arbitrairy number of tries to read a message */
```

Comment:
On 08-02-15 14:10, marcelloceschia wrote:
>
> In src/sccp_socket.c <https://github.com/marcelloceschia/chan-sccp-b/pull/254#discussion_r24298408>:
>
> > @@ -457,11 +491,12 @@ static int sccp_read_data(sccp_session_t * s, sccp_msg_t * msg)
> >  
> >  	errno = 0;
> >  	int tries = 0;
> > -	int max_retries = 30;											/* arbitrairy number of tries to read a message */
> > +	int max_retries = READ_RETRIES;											/* arbitrairy number of tries to read a message */
>
> warum max_retries noch mal definieren, wenn es eine Konstante ist.
> Macht es schwieriger zu lesen.
>
Hast recht, das kann mann entfernen. Wollte in diesem schritt so wenig wie moglich andern in der logic.
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/marcelloceschia/chan-sccp-b/pull/254/files#r24298408>.
>